network/p2p/validators.go

```diff
@@ -22,12 +22,18 @@ var (
 )
 
 type ValidatorSet interface {
-	Has(ctx context.Context, nodeID ids.NodeID) bool
+	Has(ctx context.Context, nodeID ids.NodeID) bool // TODO return error
```

Comment:
In hindsight logging the error instead of bubbling it up was a bad idea I made that I could have easily have foreseen.